BensonWollmanWeselyZombieProject/src/main/java/com/mycompany/bensonwollmanweselyzombieproject/Main.java

```diff
@@ -24,5 +26,76 @@ public static void main() {
         //Start Fight
         human1.attack_enemy(zombie1);
         zombie1.attack_enemy(human1);
+         */
+
+
+        /**
+         * Generate number of each unit and display to user
+         */
+        int numberOfHumans = (int)((Math.random() * 10) + 1);
+        int numberOfZombies = (int)((Math.random() * 10) + 1);
+
+        if (numberOfHumans > 1) {
+            System.out.println("We have " + numberOfHumans + " survivors trying to make it to safety.");
+        } else {
+            System.out.println("We have " + numberOfHumans + " survivor trying to make it to safety.");
+        }
+        if (numberOfZombies > 1) {
+            System.out.println("But there are " + numberOfZombies + " zombies waiting for them.");
+        } else {
+            System.out.println("But there is " + numberOfZombies + " zombie waiting for them.");
+        }
+        /**
+         * Create ArrayList for each unit type then generate unit objects for each type each unit type and add to it's
+         * respective ArrayList
+         */
+        ArrayList<Zombie> zombieArrayList = new ArrayList<>();
+        for (int x = 0; x < numberOfZombies; x++) {
+            zombieArrayList.add(new Zombie());
+        }
+
+        ArrayList<Survivor> survivorArrayList = new ArrayList<>();
+        for (int x = 0; x < numberOfHumans; x++) {
+            survivorArrayList.add(new Survivor());
+        }
+

```

Comment:
I think we should change zombieArrayList to work off int y instead of int x. In a few lines when we loop through zombieArrayList we use y, and I think we should keep it consistent.

Reply:
We can even stick with the more standard for of i and j too, that may be preferred

Reply:
Whatever we call the int in the for loops is fine by me.  I'm more of an x/y guy but maybe that isn't as common as it is in my own mind.

Reply:
I'm good with x and y but think it would be nice to associate x with zombies and y with survivors consistently, kind of picking knits here but I did notice it.

Reply:
I don't think it is knit picking at all, these are good suggestions to keep things consistent.